dojo/api_v2/views.py

```diff
@@ -745,6 +745,7 @@ def download_file(self, request, file_id, pk=None):
 class RiskAcceptanceViewSet(
     prefetch.PrefetchListMixin,
     prefetch.PrefetchRetrieveMixin,
+    mixins.CreateModelMixin,
     mixins.DestroyModelMixin,
     mixins.UpdateModelMixin,
     viewsets.ReadOnlyModelViewSet,
```

Comment:
This might be better (to keep it consistent)
```suggestion
    PrefetchDojoModelViewSet,
```
Please also remove `dojo_mixins.DeletePreviewModelMixin,` (I couldn't offer it because of GitHub limitations).

Reply:
https://github.com/DefectDojo/django-DefectDojo/pull/9415/commits/f0b14f435c1c35d2b29ed3cc99f28cc58b41c16a solved this